bazel/external/requirements.txt

```diff
@@ -1632,3 +1632,6 @@ setuptools==49.6.0 \
     #   -r requirements.in
     #   grpcio-tools
     #   jsonschema
+
+sdnotify==0.3.2 \
```

Comment:
Could we move this above line 1627, just to avoid any confusion? This should have been auto-generated there.